README.md

```diff
@@ -0,0 +1,38 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
I suggest to recreate this project with TypeScript. It might be a bit difficult to comprehend this language in the beginning, especially coming from python, but in long run it is a must have for such projects